test/unit/util_color.js

```diff
@@ -0,0 +1,62 @@
+var test = require('tap').test;
+var color = require('../../src/util/color');
+
+test('decimalToHex', function (t) {
+    t.equal(color.decimalToHex(0), '#000000');
+    t.equal(color.decimalToHex(1), '#000001');
+    t.equal(color.decimalToHex(16777215), '#ffffff');
+    t.equal(color.decimalToHex(-16777215), '#000001');                  // @todo
```

Comment:
The feature allowing an integer input to a `set color` block is a bit controversial, but as long as we're keeping it I think we want this behavior (including -16777215 => `#000001`). While people might not intentionally put negative numbers into that block very often, it's easy for something like this to trigger such behavior:
![image](https://cloud.githubusercontent.com/assets/7019101/19054888/4323ce92-8976-11e6-9ea0-f334c7c0938c.png)